tests/unit/test_task_registry.py

```
@@ -0,0 +1,224 @@
+        await asyncio.sleep(0)
+        assert len(registry) == 1
+
+        # task1 も完了させる
+        event.set()
+        await task1
+        await asyncio.sleep(0)
+        assert len(registry) == 0
+
+    @pytest.mark.asyncio
+    async def test_register_task_with_return_value(self) -> None:
+        """Any 型のタスク（戻り値あり）を register できる。"""
```

Ruff (0.15.10)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)